website/source/docs/agent/options.html.markdown

```diff
@@ -339,6 +349,98 @@ The remaining keys in the list are considered secondary and are used for
 decryption only. During message decryption, Serf uses the configured encryption
 keys in the order they appear in the keyring file until all keys are exhausted.
 
+#### Memberlist Profile Configuration Key Reference
+
+The following configuration keys may be used in a configuration file referenced
+using the `profile_path` option in Serf configuration.
+
+Options with a `timeout`, `time` or `interval` suffix may be expressed using
+common unit suffixes for time units - for example `500ms`, `10s` and `1m`.
+  
+* `awareness_max_mult` - Increases the probe interval if the node becomes aware
+  that it might be degraded and not meeting the soft real time requirements to
+  reliably probe other nodes.
```

Comment:
Do you think it makes sense to include the defaults for these values here? We do that with the other config docs but it's a little weird here because they come from the memberlist repo.

Reply:
We could include defaults here, but the defaults vary depending on the profile. Given that these are somewhat advanced options, I'm not sure it doesn't make sense to include a link to the relevant part of the memberlist codebase instead?

Reply:
That sounds like a good alternative 👍